r_scripts/data_wrangle/2013_oregon_case_crossover.R

```diff
@@ -0,0 +1,154 @@
+# ------------------------------------------------------------------------------
+# Title: Case crossover for diseases (Oregon State) for server
+# Author: Jingyang Liu
+# Date Created: Nov 27, 2017
+# R version: 3.3.2
+# ------------------------------------------------------------------------------
+
+#Creating case crossover dataframes
+library(dplyr) 
+library(data.table)
+library(readxl)
+library(readr)
+getwd()
+setwd("../../../data/data_new/health/")
+
+### ---------------------------------------------------------------------
+var_list <- c('saba', 'respiratory', 'asthma', 'pneumonia',  'acute_bronch', 'copd', 
+              'cvd', 'isch_heart_dis', 'arrhythmia', 'heart_failure', 
+              'cerbrovas_dis', 'myocardial_infarc', 'broken_arm')
+n <- NA
+
+start <- Sys.time()
+for (i in var_list){
+  read_path <- paste('2013_oregon', i, 'claims.csv', sep='_')
+  or_disease <- read_csv(read_path)
+  
+  or_disease1 <- or_disease %>% 
```

Comment:
this first subset of or_disease1 seems to never be used again. Can it be removed?